### ISSUE TITLE
Last line includes CR LF (\r\n) if file ends in empty line

### ISSUE THREAD

**Issue by joelverhagen** [mgholam/fastCSV#8]
I am analyzing many CSV parsers in .NET and I have noticed that fastCSV includes a `\r\n` sequence in the last column of the last row if the file ends in an empty line. Many tools include the trailing line ending (both Windows and Linux) so this behavior may be unexpected.

For example,
```
a,b,c
d,e,f

``` 
The `f` field will contain a `\r\n` after the `f`.

All of these other CSV readers do not include the `\r\n` in the last field.
![image](https://user-images.githubusercontent.com/94054/100556455-6c924180-3257-11eb-8f4c-61eb240d6106.png)

Repro: [CsvRepro.zip](https://github.com/mgholam/fastCSV/files/5613100/CsvRepro.zip)

**Comment by mgholam** [comment]
I can't reproduce the problem, do you have an example?

**Comment by joelverhagen** [comment]
Here's a small repro:
```csharp
using System;
using System.Collections.Generic;
using System.IO;
using Newtonsoft.Json;

namespace CsvReproSimple
{
    class Program
    {
        static void Main(string[] args)
        {
            var data = "a,b\r\n";
            
            using (var reader = new StringReader(data))
            {
                var nreco = new NReco.Csv.CsvReader(reader);
                var result = new List<List<string>>();
                while (nreco.Read())
                {
                    var record = new List<string>();
                    for (var i = 0; i < 2; i++)
                    {
                        record.Add(nreco[i]);
                    }
                    result.Add(record);
                }
                
                Console.WriteLine("NReco:");
                Console.WriteLine(JsonConvert.SerializeObject(result, Formatting.Indented));
            }

            Console.WriteLine();

            using (var reader = new StringReader(data))
            {
                var result = fastCSV.ReadStream<List<string>>(reader, 2, ',', (o, cols) =>
                {
                    for (var i = 0; i < 2; i++)
                    {
                        o.Add(cols[i]);
                    }
                    return true;
                });

                Console.WriteLine("fastCSV:");
                Console.WriteLine(JsonConvert.SerializeObject(result, Formatting.Indented));
            }
        }
    }
}
```

The output looks like this:
<pre>
NReco:
[
  [
    "a",
    "b"
  ]
]

fastCSV:
[
  [
    "a",
    <b>"b\r\n"</b>
  ]
]
</pre>

**Comment by mgholam** [comment]
Should be fixed in `v2.0.8`

**Comment by joelverhagen** [comment]
Awesome! I should be able to give it a try later today. I'm working on a blog post to compare performance of various CSV readers
